core/sections/clause_13_advanced_strokes.adoc

```diff
@@ -23,6 +31,10 @@ dashPattern property
 
 ===== Class Color
 
+* r: float
```

Comment:
Here we just want to use Color to type any new property. The Color class is already defined in 8-vector and 9-coverage.

Reply:
removed the associated properties